go.sum

```diff
@@ -223,8 +223,9 @@ github.com/chzyer/logex v1.1.10/go.mod h1:+Ywpsq7O8HXn0nuIou7OrIPyXbp3wmkHB+jjWR
 github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e/go.mod h1:nSuG5e5PlCu98SY8svDHJxuZscDgtXS6KTTbou5AhLI=
 github.com/chzyer/test v0.0.0-20180213035817-a1ea475d72b1/go.mod h1:Q3SI9o4m/ZMnBNeIyt5eFwwo7qiLfzFZmjNmxjkiQlU=
 github.com/client9/misspell v0.3.4/go.mod h1:qj6jICC3Q7zFZvVWo7KLAzC3yx5G7kyvSDkc90ppPyw=
-github.com/cloudflare/circl v1.3.3 h1:fE/Qz0QdIGqeWfnwq0RE0R7MI51s0M2E4Ga9kq5AEMs=
 github.com/cloudflare/circl v1.3.3/go.mod h1:5XYMA4rFBvNIrhs50XuiBJ15vF2pZn4nnUKZrLbUZFA=
```

Comment:
this seems like some dep is still using v1.3.3?

Reply:
Oh apparently some hashes are [not removable](https://go.dev/ref/mod#minimal-version-selection) in Go's module system without a `replace`, common [SO question](https://stackoverflow.com/a/70006832/3431180). It is using v1.3.7, but it detects v1.3.3 first and so still stores the hash.

This will only be removed once there is a new `go-git` version released with https://github.com/go-git/go-git/pull/988